packages/data-point/lib/entity-types/entity-request/factory.js

```diff
@@ -1,50 +1,43 @@
 const _ = require('lodash')
-const fp = require('lodash/fp')
 const createBaseEntity = require('../base-entity').create
 const createReducer = require('../../reducer-types').create
-const TransformKeys = require('./transform-keys')
 
 /**
  * @class
+ * @property {string} url
+ * @property {reducer} options
+ * @property {reducer} beforeRequest
  */
 function EntityRequest () {
   this.url = undefined
-  this.options = {}
-  this.beforeRequest = undefined
```

Comment:
I removed `this.beforeRequest` to make it optional

Reply:
done! https://github.com/ViacomInc/data-point/pull/163/commits/64813b4112dcdfd4eabc87356653e4bd72d45159